packages/hoc-react-loader-examples/package.json

```diff
@@ -48,7 +48,7 @@
     "webpack-dev-server": "~1.12.1"
   },
   "dependencies": {
-    "hoc-react-loader": "file:../",
+    "hoc-react-loader": "~6.1.1",
```

Comment:
Here, will `lerna` detect that hoc-react-loader is in a relative package and do the link on its own ?

Reply:
Sure

Reply:
And what if I upgrade the `hoc-react-loader` package to 7.0.0 without publishing it ?
What will do lerna ? Still doing a link, or download `hoc-react-loader` from NPM registry ?

Reply:
There is an option to force [local linking](https://github.com/lerna/lerna#--force-local) regardless of package version. I guess without this option it will download it from NPM registry.